src/near_indexer/Cargo.toml

```diff
@@ -21,7 +21,7 @@ serde_json = "1.0.128"
 reqwest = { version = "0.12.4", features = ["json", "multipart"] }
 regex = "1.11.0"
 near-client = "0.7.2"
-verity-client = { git = "https://github.com/usherlabs/ccamp", rev = "be304188f69019fdb11ca305cb080749a468f9db" }
+verity-client = { git = "https://github.com/usherlabs/verity-dp.git", branch = "feature/better-workspaces" }
```

Comment:
No need to add this `branch`.

Furthermore, always use `rev` not `branch`

Reply:
Okay then. I simply followed the example you shared in the issue